docs/source/explanation/algorithms/action/action_classification.rst

```diff
@@ -1,14 +1,47 @@
-Action Classificaiton
-=====================
+Action Recognition

```

Comment:
We normally use action recognition term to call both action classification and action detection. Therefore this should be action classification

Reply:
agree